float64 timestamp
float64 dBm1
float64 dBm2